lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -515,14 +644,40 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
         myLayer = myLayer[0];
         if (myLayer && myLayer.Style && myLayer.Style.length > 0) {
           for (var j = 0; j < myLayer.Style.length; ++j) {
-            if (!defined(this.isNcWMS) && myLayer.Style[j].Name && myLayer.Style[j].Name.match(/boxfill\/rainbow/i)) {
+            if (!defined(this.isNcWMS) && myLayer.Style[j].Name &&
+                (myLayer.Style[j].Name.match(/boxfill\/rainbow/i) ||
+                 myLayer.Style[j].Name.match(/default-scalar\/default/i) ||
+                 myLayer.Style[j].Name.match(/default-vector\/default/i))) {
               this.isNcWMS = true;
             }
           }
         }
       }
     }
 
+    if (!defined(this.supportsColorScaleRange)) {
+        this.supportsColorScaleRange = this.isNcWMS;
+
+        if (!this.supportsColorScaleRange) {
+            var hasExtendedRequests = capabilities.Capability &&
+                capabilities.Capability.ExtendedCapabilities &&
+                capabilities.Capability.ExtendedCapabilities &&
```

Comment:
I admire your desire to make this super thorough! But probably don't need the extra check for `capabilities.Capability.ExtendedCapabilities` :-)